test/auction-rpc-test.js

```diff
@@ -220,18 +229,110 @@ describe('Auction RPCs', function() {
   it('should create BID with signing paths', async () => {
     // Create loser's BID.
     await util.wrpc('selectwallet', [loser.id]);
-    assert(await util.wrpc('sendbid', [name, 4, 10]));
+    assert(
```

Comment:
It should be fine to have this as one liner when it's under 80.

Reply:
👍